reactor-core/src/main/java/reactor/core/publisher/MonoProcessor.java

```diff
@@ -195,14 +236,47 @@ else if(subscription == null) {
 		}
 	}
 
+	/**
+	 * Returns the internal state from -1 Cancelled to 5 errored, beyond 3 included is
+	 * fulfilled.
+	 *
+	 * @return the internal state from -1 Cancelled to 5 errored, beyond 3 included is
+	 * fulfilled.
+	 * Internal - Should not use (to be removed in 3.2)
+	 */
+	@Override
+	public long getAsLong() {
```

Comment:
just put a comment along each int magic number to explicitly tell what state it correspond to, so that future us don't get crazy reading that code 😛